cocos/gfx/base/define.ts

```diff
@@ -1671,10 +1702,7 @@ export class TextureBarrierInfo {
         public prevAccesses: AccessFlags = AccessFlagBit.NONE,
         public nextAccesses: AccessFlags = AccessFlagBit.NONE,
         public type: BarrierType = BarrierType.FULL,
-        public baseMipLevel: number = 0,
-        public levelCount: number = 1,
-        public baseSlice: number = 0,
-        public sliceCount: number = 1,
+        public range: ResourceRange = new ResourceRange(),
```

Comment:
This is also a break change.
If we have to change these, we need to add the breaking changes in 3.8.5 release note.

Reply:
Synchronization related features are not available in web. It is only used by vulkan.
I think it will not break the compatibility.